torch/nn/quantized/modules/conv.py

```diff
@@ -14,9 +14,48 @@
 
 
 class Conv2d(_ConvNd):
+    r"""Applies a 2D convolution over a quantized input signal composed of
+    several quantized input planes.
+
+    For details on input arguments, parameters, and implementation see
+    :class:`~torch.nn.quantized.Conv2d`.
```

Comment:
you mean `torch.nn.Conv2d` here, but you wrote `quantized` which is circular reference